setup/include/inc.setup_header.php

```diff
@@ -141,7 +141,6 @@
 
 // make instance of structure reader
 $ilCtrlStructureReader = new ilCtrlStructureReader();
-$ilCtrlStructureReader->setErrorObject($ilErr);
```

Comment:
Ah, k, this was the location is was asking for in `ilCtrlStructureReader`. Is `$ilErr` initialized anywhere?